testsuite/features/step_definitions/command_steps.rb

```diff
@@ -879,6 +879,10 @@
     cmd = "apt-get --assume-yes install #{package}"
     successcodes = [0]
     not_found_msg = 'Unable to locate package'
+  elsif host.include? 'slemicro'
```

Comment:
Instead of doing `host.include? 'slemicro'`, I think it will be better for homogeneity to follow the same strategy followed above, so having a method `slemicro_host?(host)`

Reply:
Nice catch. That code was from a couple of months ago. I looks much better the way you recommended it.